scripts/core/editor3/actions/editor3.jsx

```diff
@@ -0,0 +1,21 @@
+export function onChangeEditorState(editorState) {
```

Comment:
If you prefer your version, then maybe call it `changeEditorState` because it better indicates an action. `onChange` sounds like an event.